healpy/fitsfunc.py

```diff
@@ -135,7 +124,12 @@ def write_map(
     extra_header=(),
     overwrite=False,
 ):
-    """Writes a healpix map into a healpix file.
+    """Writes a healpix map into a healpix FITS file.
+
+    WARNING: starting from healpy 1.15.0, if you do not specify `dtype`,
+    the map will be written to disk with the same precision it is stored in memory.
+    Previously, by default `healpy` wrote maps in `float32`.
+    To reproduce the same behaviour of `healpy` 1.14.0 and below, set `dtype=np.float32`.
```

Comment:
```suggestion
    .. warning:: Starting from healpy 1.15.0, if you do not specify `dtype`,
       the map will be written to disk with the same precision it is stored in memory.
       Previously, by default `healpy` wrote maps in `float32`.
       To reproduce the same behaviour of `healpy` 1.14.0 and below, set `dtype=np.float32`.
```

Also change similar text from other docstrings to ReStructuredText admonitions so that they are given emphasis.

Reply:
see ca0473f863a3d299e0e9b4d07fffa3e8df547218